tests/sentry/api/test_organization_missing_members.py

```diff
@@ -43,9 +45,20 @@ def setUp(self):
         self.create_commit(repo=self.repo, author=self.nonmember_commit_author1)
         self.create_commit(repo=self.repo, author=self.nonmember_commit_author2)
 
+        not_common_domain_author = self.create_commit_author(
+            project=self.project, email="a@exampletwo.com"
+        )
+        not_common_domain_author.external_id = "not"
+        not_common_domain_author.save()
+        self.create_commit(repo=self.repo, author=not_common_domain_author)
+
         self.login_as(self.user)
 
-    def test_simple(self):
+    def test_simple__common_domain(self):
```

Comment:
This means common as in users all have the same domain right? I feel like it could be confusing since this could also refer to a test where users have "common" domains in the sense that they are seen a lot - gmail.com would be an example. Could we change the terminology here to something like `shared_domain`? or something?